.github/workflows/Integration.yml

```
@@ -0,0 +1,111 @@
+        min-coverage-changed-files: 60
+
+
+    - name: Notify Slack on Success
+      if: success() # 이 step은 job이 성공한 경우에만 실행된다.
+      id: slack-success
+      uses: slackapi/slack-github-action@v1.24.0
+      with:
+        payload: |
+          {
+            "channel": "report",
+            "attachments": 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)